tests/Resources/config/packages/db_tools_alt2.yaml

```diff
@@ -0,0 +1,28 @@
+db_tools:
+    storage:
+        root_dir: '%kernel.project_dir%/var/backup'
+        filename_strategy:
+            connection_two: app.db_tools.custom_filename_strategy
+    backup_expiration_age: '6 months ago'
```

Comment:
This value should go into `storage.expires_after` maybe ?

Reply:
We could move it in the storage section, indeed, but I would prefer to keep its current name.

However, it's not essential for now. If we decide to make this change, I propose to plan it for a future version.

Reply:
Yeah, just leave it as-is, but it's a direct storage manager parameter in the end, I think it would make sense to move it into the storage section.